nvi-test/src/main/java/no/sikt/nva/nvi/test/ExpandedAffiliation.java

```diff
@@ -0,0 +1,69 @@
+package no.sikt.nva.nvi.test;
+
+import static java.util.Objects.nonNull;
+import static no.unit.nva.testutils.RandomDataGenerator.objectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import java.net.URI;
+import java.util.Map;
+
+public record ExpandedAffiliation(URI id, String countryCode, Map<String, String> labels) {
+    public static Builder builder() {
+        return new Builder();
+    }
+
+    public static Builder from(ExpandedAffiliation other) {
+        return new Builder(other);
+    }
+
+    public ObjectNode asObjectNode() {
+        var affiliationNode = objectMapper.createObjectNode();
+
+        if (nonNull(id)) {
+            affiliationNode.put("id", id.toString());
+        }
+        if (nonNull(countryCode)) {
+            affiliationNode.put("countryCode", countryCode);
+        }
+        if (nonNull(labels)) {
+            var labelsNode = objectMapper.createObjectNode();
+            labels.forEach(labelsNode::put);
+            affiliationNode.set("labels", labelsNode);
+        }
+
+        return affiliationNode;
+    }
+
+    public static final class Builder {
+
+        private URI id;
+        private String countryCode = "NO";
```

Comment:
I think it is nice to use public constants for "hardcoded" values in project. In the case we need to update them in future, only single change will be needed, instead of updating all over the project :)

Reply:
@LarsV123 If you make these public, maybe extract to another final class?